subiquity/common/apidef.py

```diff
@@ -253,12 +255,18 @@ class has_bitlocker:
             def GET() -> List[Disk]: ...
 
         class v2:
+            class deprecated:
```

Comment:
I think the fact that we have /storage/guided /storage/v2/guided and /storage/v2/deprecated/guided is getting a little silly now... hopefully we can clean all this up soon.

Reply:
@jpnurmi - is the deprecated v2 guided method still useful?  perhaps after you do the merge we can drop this?

Reply:
the deprecated API can be removed right away but a separate PR would be much appreciated to let us update subiquity and redo the guided partitioning UI in separate steps.